src/components/common/form/message-item-types/DatesMoved.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { formatDateNoYear } from '../../../../utils/time-entry-utils/timeEntryUtils';
+import { useTimecardContext } from '../../../../context/TimecardContext';
+
+const DatesMoved = ({ variables }) => {
```

Comment:
Now that we only have one prop passed in, does it make sense to pass in startDate and endDate directly?